Shell/Options.cpp

```diff
@@ -1307,6 +1307,13 @@ void Options::init()
     _integerInductionInterval.reliesOn(Or(_induction.is(equal(Induction::INTEGER)),_induction.is(equal(Induction::BOTH))));
     _lookup.insert(&_integerInductionInterval);
 
+    _intInductionStrictness = UnsignedOptionValue("int_induction_strictness","intindst",11);
+    _intInductionStrictness.description = "Encodes term/literal combinations that are excluded from int induction. See InductionHelper.cpp for explanation.";
```

Comment:
Should I also add the complete explanation of the values here?

Reply:
If you have the time/inclination that would be nice from an end-user perspective! Probably just copying in what you already have in a comment would be fine.

Reply:
Done.